homeassistant/components/roborock/time.py

```diff
@@ -27,6 +27,19 @@
 PARALLEL_UPDATES = 0
 
 
+def _safe_time_from_cache(cache: AttributeCache, which: str) -> time:
```

Comment:
What changed in the API that required this change?

In your `_fake_get_from_cache` i only ever see it returning dicts for example. When is a non-dict is now returned here?

It seems like the simplest change would be to just handle this in `native_value` so it doesn't call `get_value` after getting the cache value and it snot a valid value.